test/helpers/increaseTime.js

```diff
@@ -43,5 +43,7 @@ export const duration = {
   minutes: function(val) { return val * this.seconds(60) },
   hours:   function(val) { return val * this.minutes(60) },
   days:    function(val) { return val * this.hours(24) },
-  weeks:   function(val) { return val * this.days(7) }
+  weeks:   function(val) { return val * this.days(7) },
+  months:  function(val) { return val * this.days(30)},
```

Comment:
Yes, it makes a total sense to mimic the solidity constants. Thanks for pointing that out!